src/main/java/com/coherensolutions/training/automation/java/web/urnezaite/BasePage.java

```diff
@@ -0,0 +1,40 @@
+package com.coherensolutions.training.automation.java.web.urnezaite;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class BasePage {
+    public WebDriver driver;
+
+    @FindBy(xpath = "//a[@title = 'View my shopping cart']")
+    WebElement cartButton;
+
+    @FindBy(css = "a.logout")
+    WebElement logOutButton;
+
+    @FindBy(xpath = "//a[@title = 'View my customer account']")
+    WebElement myAccountButton;
```

Comment:
Do not forget about encapsulation. These fields should have private access modifier